app/models/order_item.rb

```diff
@@ -91,4 +91,14 @@ def mark_finished
     self.state = 'Completed'
     save!
   end
+
+  def save
+    self.context = ManageIQ::API::Common::Request.current.to_h
+    super
+  end
+
+  def save!
+    self.context = ManageIQ::API::Common::Request.current.to_h
+    super
+  end
```

Comment:
Seems like `before_create` instead of overriding `save` methods.

Reply:
@lindgrenj6 
How about setting it in set_defaults
https://github.com/ManageIQ/service_portal-api/blob/c7fedb33c58f87ca162c307c7442e5f006cd599a/app/models/order_item.rb#L39